src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub struct Frame<T> {
 }
 
 impl<T: Copy> Frame<T> {
-    pub fn frame_as_vec3a(&self) -> impl Iterator<Item = Vec3> + '_
+    pub fn frame_as_vec3a(&self) -> impl Iterator<Item = [f32; 3]> + '_
```

Comment:
This should probably be renamed for clarity, since it's not actually a Vec3A anymore

Reply:
Yea I also noticed this and I'm working on it now.

Reply:
I just made it `impl IntoIterator` instead.